.postgraphilerc.js

```diff
@@ -10,6 +10,7 @@ module.exports = {
     cors: true,
     dynamicJson: true,
     jwtSecret: 'devsecret',
+    // disableQueryLog: true,
```

Comment:
is this to indicate that we have this option ?

Reply:
Yeah, pretty much. Can just un-comment this line whenever we want to turn it on.